Linter now flagging issue with test file

cmd/root_test.go:16:15: unused-parameter: parameter 't' seems to be unused, consider removing or renaming it as _ (revive)
func TestRoot(t *testing.T) {

--- a/cmd/root_test.go
+++ b/cmd/root_test.go
@@ -13,6 +13,6 @@ import (
 	"github.com/robertwtucker/spt-util/cmd"
 )
 
-func TestRoot(t *testing.T) {
+func TestRoot(_ *testing.T) {
 	cmd.Execute()
 }

